docs/guided-tour/target-maps/05-other-examples/README.md

```diff
@@ -0,0 +1,77 @@
+# Other Target Map Examples
+
+## Target Map on Subinstallation Level
+
+The example defined [here](../04-forward-map) deploys the same config maps on the target cluster as the others. The
+difference is that it imports 3 targets as standard targets and in this 
+[Subinstallation](../04-forward-map/component/blueprint/sub/subinst.yaml) it converts these into a target map.
```

Comment:
Just a proposal: should we move this readme into the 04 example? I was a bit confused why 04 had no readme.

Reply:
I have mode the part of the readme.